### PR TITLE
Updated to the latest QDK.

### PR DESCRIPTION
**Pull request by kuzminrobin**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "rollForward": "latestMinor"
     },
     "msbuild-sdks": {
-        "Microsoft.Quantum.Sdk": "0.24.210051-beta"
```
Comment (ricardo-espinoza) [inline global.json:7]: We should make sure that this change happens across all repos in which we added a global.json file this release.

**Review by ricardo-espinoza — APPROVED** [review]

File: global.json (hunk 1 above)
Comment (tcNickolas) [reply global.json:7]: Good point, btw - I missed that there's a beta version at https://github.com/microsoft/QuantumKatas/blob/main/global.json. We'll need to update the update scripts to keep track of this one

Comment (kuzminrobin) [reply global.json:7]: I'm doing that in all the [QDK repos](https://ms-quantum.visualstudio.com/Quantum%20Program/_wiki/wikis/Quantum-OKRs.wiki/2320/QDK-Release-Driver-Notes?anchor=qdk-repos), including qdk-private.

Comment (kuzminrobin) [reply global.json:7]: Fixing the global.json in Katas: https://github.com/microsoft/QuantumKatas/pull/788

**Pull request merged**